transport_nantes/authentication/models.py

```diff
@@ -39,6 +41,17 @@ def __str__(self):
             conf=confirmed, auth=auth,
             commune=self.commune, cp=self.code_postal)
 
+    def get_all_send_records(self) -> list:
```

Comment:
The authentication model shouldn't know anything about send records or the TB module without some extensive comment that explains why we really couldn't get around it.